tests/components/husqvarna_automower/test_switch.py

```diff
@@ -74,6 +74,12 @@ async def test_switch_commands(
         service_data={"entity_id": "switch.test_mower_1_enable_schedule"},
         blocking=True,
     )
+
+    if aioautomower_command == "parked_until_further_notice":
```

Comment:
Please don't include conditions in tests.

Reply:
Done in https://github.com/home-assistant/core/pull/110139/commits/3f96a95d0a2537c795b5fd443a0b1db84b60b54f